en/docs/develop/configure-my-account.md

```diff
@@ -0,0 +1,404 @@
+# Configuring My Account Application
+
+The **My Account** application can be configured by modifying the `deployment.config.json` file. This file can be found at the root of the app directory (`$IS_SERVER/repository/deployment/server/webapps/myaccount`) in production and in `apps/myaccount/src/public/`  in the dev environment.
+
+By default, the `deployment.config.json` file has the following configurations.
+
+```json
+{
+	"appBaseName": "myaccount",
+	"clientID": "USER_PORTAL",
+	"loginCallbackPath": "/login",
+	"logoutCallbackPath": "/login",
+	"productVersion": "some",
+	"serverOrigin": "https://localhost:9443",
+	"routePaths": {
+		"home": "/overview",
+		"login": "/login",
+		"logout": "/logout"
+	},
+	"session": {
+		"userIdleTimeOut": 600,
+		"userIdleWarningTimeOut": 580,
+		"sessionRefreshTimeOut": 300,
+		"checkSessionInterval": 3
+	},
+	"ui": {
+		"appCopyright": "WSO2 Identity Server",
+		"appTitle": "My Account | WSO2 Identity Server",
+		"appName": "My Account",
+		"appLogoPath": "/assets/images/logo.svg",
+		"authenticatorApp": {
+			"enabled": true
+		},
+		"features": {
+			"applications": {
+				"disabledFeatures": [],
+				"enabled": true,
+				"scopes": {
+					"create": [],
+					"read": [],
+					"update": [],
+					"delete": []
+				}
+			},
+			"operations": {
+				"disabledFeatures": [],
+				"enabled": true,
+				"scopes": {
+					"create": [],
+					"read": [ "internal_humantask_view" ],
+					"update": [],
+					"delete": []
+				}
+			},
+			"overview": {
+				"disabledFeatures": ["overview.accountStatus"],
+				"enabled": true,
+				"scopes": {
+					"create": [],
+					"read": [],
+					"update": [],
+					"delete": []
+				}
+			},
+			"personalInfo": {
+				"disabledFeatures": [],
+				"enabled": true,
+				"scopes": {
+					"create": [],
+					"read": [],
+					"update": [],
+					"delete": []
+				}
+			},
+			"security": {
+				"disabledFeatures": [],
+				"enabled": true,
+				"scopes": {
+					"create": [],
+					"read": [],
+					"update": [],
+					"delete": []
+				}
+			}
+		},
+		"productName": "Identity Server",
+		"productVersionConfig": {
+			"allowSnapshot": true,
+			"textCase": "uppercase",
+			"labelColor": "primary"
+		},
+		"theme": {
+			"name": "default"
+		}
+	}
+}
+```
+## Configure basic settings
```

Comment:
shall we use the line separator between subheadings? Please check throughout all the content